packages/xrpl/test/utils/hashLedger.test.ts

```diff
@@ -10,7 +10,7 @@ const { hashLedger: REQUEST_FIXTURES } = requests
 
 describe('hashLedger', function () {
   let ledger
-  beforeEach(function () {
+  beforeEach(() => {
```

Comment:
ditto - seems easy enough to undo with a ctrl-f-replace

Reply:
Done